src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -1761,6 +1794,28 @@ private void checkOutputPaths() {
     }
   }
 
+  private Map<String, Object> getArrayAttributes() throws InterruptedException {
+    Map<String, Object> attrs = new HashMap<String, Object>();
+    String order = null;
+    if (dimensionOrder != null) {
+      order = dimensionOrder.toString();
+    }
+    else {
+      IFormatReader reader = readers.take();
+      try {
+        order = reader.getDimensionOrder();
+      }
+      finally {
+        readers.put(reader);
+      }
+    }
+
+    String[] axes =
```

Comment:
As mentioned yesterday and as per https://github.com/ome/ngff/pull/96, this block is no longer a requirement of the specification (and is effectively broken in the default layout) so we can safely remove it.